webpki-root-certs/tests/codegen.rs

```diff
@@ -0,0 +1,82 @@
+use std::fmt::Write;
+use std::fs;
+
+use ring::digest;
+
+use webpki_ccadb::fetch_ccadb_roots;
+
+#[tokio::test]
+async fn new_generated_code_is_fresh() {
+    let tls_roots_map = fetch_ccadb_roots().await;
+    let mut code = String::with_capacity(256 * 1_024);
+    code.push_str(HEADER);
+    code.push_str("pub const TLS_SERVER_ROOT_CERTS: &[CertificateDer] = &[\n");
```

Comment:
We should make the lifetime explicit, IMO.

Reply:
I've added the elided lifetime here, and on `TLS_SERVER_ROOTS` in `webpki-roots`. I also added `elided_lifetimes_in_paths` to the generated `lib.rs` deny list so we don't regress. The `webpki-root-certs` fixes were rolled into the initial commit and the `webpki-roots` updates are done in a separate commit.